src/components/POLine/Cost/CostView.js

```diff
@@ -106,7 +100,7 @@ function CostView({ cost, isPackage, orderFormat }) {
           >
             <AmountWithCurrencyField
               currency={currency}
-              amount={get(cost, 'listUnitPriceElectronic')}
+              amount={cost.listUnitPriceElectronic}
```

Comment:
will it be always defined? based on prop types it's not required

Reply:
defaultProps `cost: { }`

Reply:
I hope no `null` here as well

Reply:
🤞